xbmc/application/Application.cpp

```diff
@@ -934,6 +932,9 @@ void CApplication::Render()
   CServiceBroker::GetWinSystem()->GetGfxContext().Flip(hasRendered,
                                                        appPlayer->IsRenderingVideoLayer());
 
+  if (appPower->GetRenderGUI() && !m_skipGuiRender)
+    CServiceBroker::GetGUI()->GetWindowManager().AfterRender();
+
```

Comment:
This now seems to execute after the flip, should it be renamed BeforeRender?

Reply:
I moved it there because I can only query the buffer age after the flip. Maybe "AfterFlip"?